src/components/TiltCard.tsx

```diff
@@ -0,0 +1,192 @@
+import {
+  useRef,
+  useState,
+  useCallback,
+  forwardRef,
+  type CSSProperties,
+} from "react";
+import { classes } from "../utils/classes";
+
+interface ConditionalTiltCardProps extends TiltCardProps {
+  disabled?: boolean;
+}
+
+const ConditionalTiltCard = forwardRef<
+  HTMLDivElement,
+  ConditionalTiltCardProps
+>(function ConditionalTiltCard(
+  {
+    children,
+    disabled = true,
+    maxTilt = 15,
+    glareMaxOpacity = 0.4,
+    className = "",
```

Comment:
`className` should be fine to default to `undefined`, rather than an empty string?